.test/meta-commands/out.sh

```diff
@@ -137,6 +137,42 @@ jq -s '
 	)
 ' temp/index.json > temp/index.json.new
 mv temp/index.json.new temp/index.json
+# SBOM
+originalImageManifest="$(jq -r '.manifests[0].digest' temp/index.json)"
+SOURCE_DATE_EPOCH=1709081058 \
+	docker buildx build --progress=plain \
+	--load=false \
+	--provenance=false \
+	--build-arg BUILDKIT_DOCKERFILE_CHECK=skip=all \
+	--sbom=generator="$BASHBREW_BUILDKIT_SBOM_GENERATOR" \
+	--output 'type=oci,tar=false,dest=sbom' \
+	--platform 'linux/amd64' \
+	--build-context "fake=oci-layout://$PWD/temp@$originalImageManifest" \
+	- <<<'FROM fake'
+sbomIndex="$(jq -r '.manifests[0].digest' sbom/index.json)"
+shell="$(jq -r --arg originalImageManifest "$originalImageManifest" '
+	first(
+		.manifests[]
+		| select(.annotations["vnd.docker.reference.type"] == "attestation-manifest")
+	) as $attDesc
+	| @sh "sbomManifest=\($attDesc.digest)",
+		@sh "sbomManifestDesc=\(
+			$attDesc
+			| .annotations["vnd.docker.reference.digest"] = $originalImageManifest
+			| tojson
+		)"
+' "sbom/blobs/${sbomIndex/://}")"
+eval "$shell"
+shell="$(jq -r '
+	"copyBlobs=( \([ .config.digest, .layers[].digest | @sh ] | join(" ")) )"
+' "sbom/blobs/${sbomManifest/://}")"
+eval "$shell"
+copyBlobs+=( "$sbomManifest" )
+for blob in "${copyBlobs[@]}"; do
+	cp "sbom/blobs/${blob/://}" "temp/blobs/${blob/://}"
+done
+jq -r --argjson sbomManifestDesc "$sbomManifestDesc" '.manifests += [ $sbomManifestDesc ]' temp/index.json > temp/index.json.new
+mv temp/index.json.new temp/index.json
 # </build>
 # <push>
 crane push --index temp 'oisupport/staging-amd64:191402ad0feacf03daf9d52a492207e73ef08b0bd17265043aea13aa27e2bb3f'
```

Comment:
What is the meaning of this checksum-like tag? Currently the checksum of [this image in Hub](https://explore.ggcr.dev/?image=oisupport%2Fstaging-amd64%3A191402ad0feacf03daf9d52a492207e73ef08b0bd17265043aea13aa27e2bb3f) is sha256:70a227928672dffb7d24880bad1a705b527fab650f7503c191e48a209c4a0d10

Reply:
I think this is coming from the test data so it's possible it doesn't match the actual image

Reply:
But why not just push a tag like `meta-scripts-test`?

Reply:
Yes, this is coming from the test data -- this is the "buildId" tags that we use for all our staging pushes.  It looks like a checksum because it _is_ a checksum (of https://github.com/docker-library/meta-scripts/blob/9c6f9d47a24ff6940ee3071e60f2456669f16b23/cmd/builds/main.go#L34-L38).